agent/Makefile

```diff
@@ -9,6 +9,7 @@ clean:
 
 .PHONY: get-deps
 get-deps:
+	go install mvdan.cc/gofumpt@latest
```

Comment:
can we pin this to a version for security reasons?

Reply:
sure.

Reply:
I didn't because I figured it was low risk and pinned things always just get stale until they break, but I agree security is a greater concern than that.